### PR TITLE
proj 4.9.1

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

**Comment by dunn** [comment]
I guess, uh, closes #37653 if the build passes.

**Comment by dunn** [comment]
Can't replicate the spatialite-tools failure locally.

**Comment by dunn** [comment]
OK, whittled the problems down to a SHA mismatch:
```
==> Installing liblas dependency: liblwgeom
==> Downloading https://homebrew.bintray.com/bottles/liblwgeom-2.1.5_1.mountain_lion.bottle.tar.gz
Already downloaded: /Library/Caches/Homebrew/liblwgeom-2.1.5_1.mountain_lion.bottle.tar.gz
Error: SHA1 mismatch
Expected: 6432a799ec96fd4583610f64c1ca88f3c5931fc4
Actual: e204a6ebbc36522ded9e649f949c83704e9f70ae
Archive: /Library/Caches/Homebrew/liblwgeom-2.1.5_1.mountain_lion.bottle.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by mikemcquaid** [comment]
Needs rebased on `master` now, I'm afraid. I think I've fixed the SHA mismatch though.

**Comment by dunn** [comment]
Should I have added `revision 1` to the new `gdal`?

**Comment by dunn** [comment]
Well that answers that question but the liblwgeom bottle is still acting up:
```
==> Installing liblas dependency: liblwgeom
==> Downloading https://homebrew.bintray.com/bottles/liblwgeom-2.1.5_1.yosemite.bottle.1.tar.gz
Error: Failed to download resource "liblwgeom"
Download failed: https://homebrew.bintray.com/bottles/liblwgeom-2.1.5_1.yosemite.bottle.1.tar.gz
```

I'll push the revision to gdal once that's fixed so the bot just needs to run this once more.

File: Library/Formula/libgeotiff.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Libgeotiff < Formula
   homepage 'http://geotiff.osgeo.org/'
   url 'http://download.osgeo.org/geotiff/libgeotiff/libgeotiff-1.4.1.tar.gz'
   sha1 'bc9e2bb43f3877b795b4b191e7aec6267f4a1c7e'
+  revision 1
```
Comment (mikemcquaid) [inline Library/Formula/libgeotiff.rb]: Ah, I think you need to kill the bottle block here.

Comment (dunn) [inline Library/Formula/libgeotiff.rb]: the libgeotiff bottle block, or the bottle block for liblwgeom?

Comment (mikemcquaid) [inline Library/Formula/libgeotiff.rb]: `libgeotiff` I believe, maybe everything with a new `revision` (if you check the build output)

**Comment by dunn** [comment]
Idk if burning all the bottle blocks was overkill but it's green now.

**Comment by mikemcquaid** [comment]
Thanks @dunn!

**Pull request closed without merging**